roles/edpm_network_config/tasks/main.yml

```diff
@@ -23,6 +23,27 @@
     - edpm_network_config_tool == 'nmstate'
     - not edpm_network_config_tool_nmstate_override|bool
 
+- name: Validate edpm_network_config_template against configuration tool
+  vars:
+    _tmpl: "{{ edpm_network_config_template | from_yaml }}"
+  block:
+    - name: Assert nmstate compatible edpm_network_config_template
+      when:
+        - edpm_network_config_tool == 'nmstate' or edpm_network_config_nmstate | bool
```

Comment:
Not really related to your patch, but why do we have *two* variable here instead of just one ?

Reply:
because there are two of them apparently:
- one to use nmstate directly ("protected" by the first task of this file)
- one to instruct os-net-config to use nmstate as a backend instead of ifcfg